packages/server/graphql/mutations/addTeam.ts

```diff
@@ -45,7 +45,10 @@ export default {
         .getAll(orgId, {index: 'orgId'})
         .run()
 
-      const orgTeamNames = orgTeams.map((team) => !team.isArchived && team.name)
+      const orgTeamNames = [] as string[]
+      orgTeams.forEach((team) => {
+        if (!team.isArchived) orgTeamNames.push(team.name)
+      })
       const {
         data: {newTeam},
         errors
```

Comment:
Why is `addTeamValidation` a thunk out of interest?

Reply:
me from years ago was a terrible programmer